xbmc/network/WakeOnAccess.cpp

```diff
@@ -437,28 +570,39 @@ bool CWakeOnAccess::WakeUpHost(const WakeUpEntry& server)
   return true;
 }
 
-bool CWakeOnAccess::FindOrTouchHostEntry (const std::string& hostName, WakeUpEntry& result)
+bool CWakeOnAccess::FindOrTouchHostEntry(const std::string& hostName, bool upnpMode, WakeUpEntry& result)
 {
   CSingleLock lock (m_entrylist_protect);
 
   bool need_wakeup = false;
 
+  UPnPServer* upnp = upnpMode ? LookupUPnPServer(hostName) : nullptr;
```

Comment:
No, those private lists of UpnpServers and associated functions exists in all cases ; they are just empty when compiled without HAS_UPNP